### PR TITLE
Allow users to do everything except start a server

### LINKED ISSUE #448: Change Actions a User Can Perform Without an Active Subscription
Users should be able to do everything except start a server if they don't have an active subscription.

### PR DESCRIPTION
**Pull request by johngriebel**
Closes #448
Signed-off-by: John Griebel <jgriebel@3blades.io>

### CODE REVIEW

File: billing/middleware.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def __call__(self, request):
                 customer, _ = create_stripe_customer_from_user(user)
 
             if ((not customer.has_active_subscription())
-                and (url_name not in settings.SUBSCRIPTION_EXEMPT_URLS)
+                and (url_name in settings.SUBSCRIPTION_REQUIRED_URLS)
                 and request.method != "GET"):
```
Comment (raygesualdo) [inline billing/middleware.py]: With `SUBSCRIPTION_REQUIRED_URLS`, do you still need to do the `request.method != "GET"` check?

File: servers/fixtures/server_sizes.json [hunk 2]
```diff
@@ -0,0 +1 @@
...emory": 8192, "description": "", "created_at": "2017-10-23T13:12:20.579Z", "active": true, "storage_size": null, "cost_per_second": "0.000000"}}, {"model": "servers.serversize", "pk": "d922b3b9-957b-48c6-bc1a-65d855fc7e7a", "fields": {"name": "Medium", "cpu": 1, "memory": 2048, "description": "", "created_at": "2017-10-23T13:11:43.874Z", "active": true, "storage_size": null, "cost_per_second": "0.000000"}}, {"model": "servers.serversize", "pk": "da0f5183-ce12-42a3-a739-63156e08ad75", "fields": {"name": "Small", "cpu": 1, "memory": 1024, "description": "", "created_at": "2017-10-23T13:11:36.187Z", "active": true, "storage_size": null, "cost_per_second": "0.000000"}}, {"model": "servers.serversize", "pk": "eeb394da-544d-4206-abcb-990f28475bcc", "fields": {"name": "Nano", "cpu": 1, "memory": 512, "description": "", "created_at": "2017-10-23T13:11:28.251Z", "active": true, "storage_size": null, "cost_per_second": "0.000000"}}, {"model": "servers.serversize", "pk": "efb340d6-aa9b-4961-99fb-3a8fba475b18", "fields": {"name": "Large", "cpu": 1, "memory": 4096, "description": "", "created_at": "2017-10-23T13:11:56.755Z", "active": true, "storage_size": null, "cost_per_second": "0.000000"}}]
```
Comment (raygesualdo) [inline servers/fixtures/server_sizes.json]: Why is this file included in the PR?

File: billing/middleware.py (hunk 1 above)
Comment (johngriebel) [reply billing/middleware.py]: No
